src/families/tezos/bridge/js.ts

```diff
@@ -201,6 +201,7 @@ const prepareTransaction = async (
           mutez: true,
           to: transaction.recipient,
           amount: amount.toNumber(),
+          storageLimit: 257, // https://github.com/TezTech/eztz/blob/master/PROTO_003_FEES.md for originating an account
```

Comment:
can you use a constant from taquito instead? like we did for `DEFAULT_FEE.REVEAL`
probably this: https://github.com/ecadlabs/taquito/blob/48a11cfaffe4c6bdfa6f04ebbd0b756f4b135865/packages/taquito/src/constants.ts#L15